tutorials/poststack_nccl.py

```diff
@@ -0,0 +1,197 @@
+r"""
+Post Stack Inversion - 3D with Nvidia's NCCL
+=========================
+This tutorial is similar to poststack.py but it shows how to run PyLops-MPI in multi-GPU environment
```

Comment:
This tutorial is similar to poststack.py but it shows how to run PyLops-MPI in multi-GPU environment and have those GPUs communicate via NCCL -> This tutorial is an extension of the poststack.py tutorial where PyLops-MPI is run in multi-GPU setting with GPUs communicating via NCCL.